dev/components/stocks.js

```diff
@@ -0,0 +1,81 @@
+import $ from 'jquery';
+import Graph from '../components/graph.js';
+
+class Stocks {
+  constructor(container) {
+
+    // REGISTER ELEMENTS
+    this.$container = container;
+
+    this.renderStocksPageHTML();
+
+    this.$stockListContainer = $('#stocks-list');
+
+    // INITIALIZE FETCH OF STOCKS
+    this.getStocks();
+  }
+
+
+  renderStocksPageHTML() {
```

Comment:
haaha, very literal name of a function. Perhaps just `render`; 

the maybe `renderStocks` too